content/support.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Support"
+description = "Support for deploying and using KEDA"
++++
+
+## OSS support
+KEDA is an open-source project under CNCF Foundation and provides best-effort to support its community of users.
+
+We have a nice community that is always happy to help each other. We support our users either via GitHub discussions or [#Keda](https://kubernetes.slack.com/messages/CKZJ36A5D) channel on the [Kubernetes slack](https://slack.k8s.io/). We use GitHub for tracking bugs and feature requests.
```

Comment:
I'd align a bit with our support policy and point to it:
```suggestion
## Open-Source (OSS) support
KEDA is an open-source project under CNCF Foundation and provides best-effort support and use GitHub for tracking bugs and feature requests.

Want to contribute a feature or fix? We are more than happy to review requests and contributions, but recommend going through our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md).

Learn more in our [support policy](https://github.com/kedacore/governance/blob/main/SUPPORT.md).
```

Reply:
Fine by me

Reply:
alright